.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -2,14 +2,18 @@
 name: Product Release Checks
 about: Use this template to track all release checks for your component
 title: "[Product] Release Checks"
-labels: compliance, documentation, "security analysis", "test results"
+labels: foss
+milestone: "24.05"
 ---
 
+> [!IMPORTANT]  
```

Comment:
Maybe you could write the full process within a little markdown itself and reference it within the template, but in my opinion to have all instructions how to use this template within the template seems a bit strange for me. 😕

Reply:
@FaGru3n sure, where could this additional markdown be stored? In the repo root?

Reply:
maybe directly on the readme?
https://github.com/eclipse-tractusx/sig-release/blob/main/README.md#release-management-acceptance-criteria

Reply:
Alright, thanks! I'll propose the change there.